### PR TITLE
feat(ui): Move "remove team" into team settings panel

### PR DESCRIPTION
**Pull request by benvinegar**
image

### CODE REVIEW

**Comment by benvinegar** [comment]
I wanted to write a test – or at least a render / snapshot test of this – but I see the spec file covers `teamSettings.old`. Is there a spec for this floating around I missed?

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx [hunk 1]
```diff
@@ -36,26 +40,51 @@ export default class TeamSettings extends AsyncView {
...
-        initialData={{
-          name: team.name,
-          slug: team.slug,
-        }}
-      >
-        <Box>
-          <JsonForm location={this.context.location} forms={teamSettingsFields} />
-        </Box>
-      </Form>
+      <div>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx]: Can we use `<React.Fragment>` instead of an extra container here?

**Comment by benvinegar** [comment]
@billyvg Could you take another look? Updated and rebased.

File: src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx [hunk 2]
```diff
@@ -52,24 +54,49 @@ export default class TeamSettings extends AsyncView {
...
+            slug: team.slug,
+          }}
+        >
+          <Box>
+            <JsonForm location={this.context.location} forms={teamSettingsFields} />
+          </Box>
+        </Form>
+
+        {access.has('team:admin') && (
+          <div className="box">
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx]: oops I missed this the first time but if copy this it'll give us a more consistent UI.

https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/organization/general/organizationGeneralSettingsView.jsx#L144

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx (hunk 2 above)
Comment (benvinegar) [reply src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx]: Like, use `<Panel/>` and `<PanelHeader/>`, etc?

Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/team/teamSettings.jsx]: Ya and `Field`

**Pull request merged**